products/green_fast_track/models/product/green_fast_track_bbls.sql

```diff
@@ -55,11 +55,20 @@ flags_wide AS (
 SELECT
     pluto.bbl,
     {% for row in variables -%}
-        CASE
-            WHEN f."{{ row['label'] }}" IS NULL THEN 'No'
-            ELSE 'Yes'
-        END AS "{{ row['label'] }} Flag",
-        f."{{ row['label'] }}",
+        {% if row['variable_type'] == 'zoning_districts' %}
+            /* zoning_districts has a Category column */
+            f."{{ row['label'] }}" AS "{{ row['label'] }} Category",
+            array_to_string(ARRAY[pluto.zonedist1, pluto.zonedist2, pluto.zonedist3, pluto.zonedist4],
```

Comment:
Does this include nulls in the array?

Reply:
oh true! this approach is definitely ugly and not DRY

maybe better if I make sure the list generated during `int_flags__zoning` is accessible here?

Reply:
Eh that list gets chopped to a million pieces. I think this is fine, just should have somethin to filter out nulls in creating the array so that users don't see "C1, Null, Null, Null"

Reply:
noting that we found that `array_to_string` ignores nulls and if their all null the default is an empty string